Makefile

```diff
@@ -113,23 +113,33 @@ submodules:
 	@git submodule sync
 	@git submodule update --init --recursive
 
-.PHONY: manifests cobertura reviewable submodules fallthrough test-integration
+# This is for running out-of-cluster locally, and is for convenience. Running
+# this make target will print out the command which was used. For more control,
+# try running the binary directly with different arguments.
+run: go.build
```

Comment:
since this is for running, should it be named `go.run`?

Reply:
@soorena776 `go.build` is defined here: https://github.com/upbound/build/blob/master/makelib/golang.mk#L152-L156

We don't have a `go.run` `Makefille` target.  If we were to add one, I imagine it could skip the tests and would avoid clobbering an existing build artifact.  Although my take on the build artifacts is that they don't take long to produce and they not expected to be long lived.

Reply:
My thinking is that `go.run` feels more like a go-specific run target, or a run target for a subsystem. In this case, by using `run` I wanted to indicate to the caller that the `run` is for the single top-level entrypoint of the system which we build with this `Makefile`.